README.md

```diff
@@ -68,3 +68,5 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```

Comment:
Great job @Joel-100 🤝

- [ ] [Optional] Kindly add the correct readme file and customize it according. You may use this [readme template](
https://github.com/microverseinc/readme-template) to get started.